### ISSUE TITLE
Ability to read .ags file format

### ISSUE THREAD

**Issue by DougiB** [agile-geoscience/striplog#158]
Would it be possible to develop striplog to read AGS file formats such as it can read LAS formats. AGS is widely used in the EU and would be highly desirable.

For more on AGS files and the formats here is a link to the web page: https://www.ags.org.uk/data-format/ags4-data-format/

**Comment by kwinkunks** [comment]
@DougiB Thank you, this seems sensible.

I'm not familiar with the format. Do you have one or more examples of such files that you are able to share?

Also, can you comment on how much variance there is between files in the wild and the specification?

**Comment by DougiB** [comment]
@kwinkunks Yeah sure, I can send you a file which is a good example of the files. I have been in contact with @mtb-za and @EvanBianco concerning this and I have already sent them over a copy of an AGS file.

Concerning the deviation of the produced files from the standard, in my experience the contractors which I have dealt with have been quite good in sticking to the default templates. There is generally no reason to deviate from the templates as they have been created by the AGS professional board for the industry.

PS: This could also be very useful for Welly.

**Comment by kwinkunks** [comment]
Here's a fairly small AGS file example:

https://webapps.bgs.ac.uk/services/ngdc/accessions/index.html?#item163546

**Comment by Didymograptus** [comment]
I think that striplog might be overkill for AGS data. AGS has fields for a legend code (normally used to show lithology type, e.g. sandstone, igneous rock, clay, gravel etc) and also a geology code that may be used to define the formation, e.g. London clay. Using these data items it is possible to simply create a striplog-type plot. That's not to say that the principles used in striplog couldn't be used to extract other qualitative data items, such as soil and rock strength. 

Tony
